### DISCUSSION TITLE
error '\deferred@thm@head' (Please help!)

### DISCUSSION THREAD

**Discussion by aleksejalex** [texstudio-org/texstudio#3410] [Q&A] [answered]
hi, 
I installed newest version of TexStudio on Linux Mint, where TexLive2023 already is installed.

When compiling my file in TexStudio, I get error: `LaTeX hooks Error: Generic hooks cannot be added to '\deferred@thm@head'`.

when compiling in terminal via `pdflatex`, everything is fine. I don't know what to do, I already reinstalled texlive2023 and texstudio several times, tried everything.

Please help!

**Comment by muzimuzhi** [comment]
By comparing log files, you may find at least one of the loaded files are different.

Without a complete example I can only guess. Also could you edit title of current discussion to be more specific?

_Update:_ Probably related to a resolved `hyperref` issue, see https://github.com/latex3/hyperref/issues/304.

**Reply by aleksejalex** [reply, marked as answer]
Thank you very much for the reply. 

I went through every single line of preamble, my only clue was potential incompatibility of packages. 
And I found it: yes, excluding hyperref seemed to solve the problem, but I need it. So I dug deeper.

And: the problem vanished when I set only one language in \usepackage[all my languages were here]{babel}. Only this solved the problem.

**Reply by muzimuzhi** [reply]
If upgrading to the latest `hyperref` won't solve it for you, then you can report this incompatibility to https://github.com/latex3/hyperref/issues.
